### PR TITLE
GUI-2288: Switch to new tag editor on security group pages

### PR DESCRIPTION
**Pull request by kamalgill**
Implements https://eucalyptus.atlassian.net/browse/GUI-2288 by incorporating new tag editor on Create Security Group page and details page

### CODE REVIEW

File: eucaconsole/views/securitygroups.py [hunk 1]
```diff
@@ -310,7 +311,8 @@ def securitygroup_create(self):
                     self.add_rules(security_group=new_security_group, traffic_type='egress')
                 if tags_json:
                     tags = json.loads(tags_json)
-                    for tagname, tagvalue in tags.items():
+                    tags_dict = self.normalize_tags(tags)
```
Comment (coyote240) [inline eucaconsole/views/securitygroups.py]: Are you sure you need to call normalize_tags?  If I remember right, it was only used in elbs to work around something wicked going on in there.  normalize_tags isn't called in other tag editor implementations.

Comment (kamalgill) [inline eucaconsole/views/securitygroups.py]: The tags come across as a list of dicts (after json.loads), so we need the normalize_tags call here.

Comment (kamalgill) [inline eucaconsole/views/securitygroups.py]: Note that we also seem to need the normalize_tags call when using the new tag editor on the Create Image from Instance page (see the GUI-2283-new WIP branch)

Comment (coyote240) [inline eucaconsole/views/securitygroups.py]: Lame.  Ok, let's merge it.

**Pull request merged**